### PR TITLE
Feature/launch page format

### PR DESCRIPTION
**Pull request by rachaelcarroll**
## Description

Update margins on main, fix search bar bug, upload logo and fix sizing, added character max length to answer form and character counter 

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Refactor

## Concerns
Next steps?
- CSS ... decide final colors/design 
- CYPRESS TESTING

### CODE REVIEW

File: src/Components/App.js [hunk 1]
```diff
@@ -52,7 +52,7 @@ export default class App extends Component {
       <>
         {/* <Header /> */}
         <NavLink to='/'>
-          <h1 className="main-title">{'<OnlyDevs />'}</h1>
+          <img className='logo' src='https://i.imgur.com/imPxyaU.png' alt='only-devs logo' id='onlyDevsLogo'></img>
```
Comment (shayan-golafshani) [inline src/Components/App.js:55]: I love the use of a custom logo, Andrew & Rach. Bravo, governors!

File: src/Components/Question.js [hunk 2]
```diff
@@ -10,6 +10,8 @@ export const Question = ({ randomQuestion, postAnswer }) => {
     const [answer, setAnswer] = useState('')
     const [isDisabled, setDisabled] = useState(true);
     const [isAnswered, setIsAnswered] = useState(false);
+    const maxLength = 300;
```
Comment (shayan-golafshani) [inline src/Components/Question.js:13]: Nice use of hooks for the char counter

File: src/Components/Question.js [hunk 3]
```diff
@@ -24,23 +26,31 @@ export const Question = ({ randomQuestion, postAnswer }) => {
         setIsAnswered(true); 
        }
 
+    useEffect(() => {
+    setCharsLeft(maxLength - answer?.length);
```
Comment (shayan-golafshani) [inline src/Components/Question.js:30]: Ah nice, a tricky bit of logic, with the optional question length argument.

File: src/Components/Question.js [hunk 4]
```diff
@@ -24,23 +26,31 @@ export const Question = ({ randomQuestion, postAnswer }) => {
...
             </header>
-            <input 
+            <textarea
+                className='answer-input' 
                 id={randomQuestion.id}
-                type='text'
                 name='answer'
                 placeholder='Write your answer here.'
+                // rows={3}
+                onKeyPress={(e) => { e.key === 'Enter' && e.preventDefault(); }}
```
Comment (shayan-golafshani) [inline src/Components/Question.js:50]: Nice prevention of submission in the onKeyPress!

File: src/Components/Question.js [hunk 5]
```diff
@@ -24,23 +26,31 @@ export const Question = ({ randomQuestion, postAnswer }) => {
...
                 value={answer}
+                autoComplete='off'
```
Comment (shayan-golafshani) [inline src/Components/Question.js:52]: Good job on removing auto-complete completely unnecessary

File: src/Components/QuestionBoard.js [hunk 6]
```diff
@@ -25,15 +25,16 @@ export const QuestionBoard = ({questions}) => {
...
-        <input className='search-bar'
-            type="text" 
-            placeholder="Search By Keyword..." 
-            onChange={(event) => {
-                setSearchTerm(event.target.value);
-                }} />
-        <section className='cards-container'>
-        {questionCard.length ? questionCard : <h2> There are no questions that match your search.</h2>} 
-        </section>
+            <input className='search-bar'
```
Comment (shayan-golafshani) [inline src/Components/QuestionBoard.js:28]: Nice, you just moved the search input to another spot, right?

File: src/styles/V-components/Question.scss [hunk 7]
```diff
@@ -18,24 +20,20 @@
...
 
 .question-header {
   @include flex(row, center)
 }
 
 .shuffle-btn {
-  width: 60px;
-  height: 35px;
+  width: 75px;
+  height: 59px;
```
Comment (shayan-golafshani) [inline src/styles/V-components/Question.scss:36]: Maybe we can this responsive for mobile

File: src/styles/V-components/Question.scss [hunk 8]
```diff
@@ -64,4 +62,9 @@
 .gold-btn {
   background: gold;
   transform: scale(1.1);
+}
+
+.char-counter {
+  display: flex;
```
Comment (shayan-golafshani) [inline src/styles/V-components/Question.scss:68]: I really like this being display on the right side of the container! Bravo

**Review by shayan-golafshani — APPROVED** [review]
This looks great Rach

File: src/Components/App.js (hunk 1 above)
Comment (andrewvallejo) [reply src/Components/App.js:55]: thanks!

**Pull request merged**